torchvision/models/googlenet.py

```diff
@@ -56,25 +60,25 @@ def __init__(self, num_classes=1000, aux_logits=True, transform_input=False, ini
 
         self.inception5a = Inception(832, 256, 160, 320, 32, 128, 128)
         self.inception5b = Inception(832, 384, 192, 384, 48, 128, 128)
-        if aux_logits:
-            self.aux1 = InceptionAux(512, num_classes)
-            self.aux2 = InceptionAux(528, num_classes)
+
+        self.aux1 = InceptionAux(512, num_classes)
+        self.aux2 = InceptionAux(528, num_classes)
+
         self.avgpool = nn.AdaptiveAvgPool2d((1, 1))
-        self.dropout = nn.Dropout(0.4)
+        self.dropout = nn.Dropout(0.2)
         self.fc = nn.Linear(1024, num_classes)
 
         if init_weights:
             self._initialize_weights()
 
     def _initialize_weights(self):
         for m in self.modules():
-            if isinstance(m, nn.Conv2d):
-                nn.init.xavier_uniform_(m.weight)
-                if m.bias is not None:
-                    nn.init.constant_(m.bias, 0.2)
-            elif isinstance(m, nn.Linear):
-                nn.init.xavier_uniform_(m.weight)
-                nn.init.constant_(m.bias, 0)
+            if isinstance(m, nn.Conv2d) or isinstance(m, nn.Linear):
+                import scipy.stats as stats
+                X = stats.truncnorm(-2, 2, scale=0.01)
+                values = torch.Tensor(X.rvs(m.weight.numel()))
```

Comment:
Can you use `torch.as_tensor` instead, by passing `dtype` to match the dtype of the weight? `torch.Tensor` as a constructor is legacy now